lib/bloc/input/input_bloc.dart

```diff
@@ -102,10 +105,12 @@ class InputBloc extends Cubit<InputState> {
     _log.info("handlePaymentRequest: $inputData source: $source");
     final LNInvoice lnInvoice = inputData.invoice;
 
-    /*NodeState? nodeState = await _breezSDK.nodeInfo();
-    if (nodeState == null || nodeState.id == lnInvoice.payeePubkey) {
+    // Ignore self payments
+    final liquidWallet = ServiceInjector().liquidSDK.wallet;
+    liquid_sdk.GetInfoResponse walletInfo = await liquidWallet!.getInfo();
+    if (walletInfo.pubkey == lnInvoice.payeePubkey) {
```

Comment:
This is wrong. Our node is not lightning node and the invoice pubkey is the swapper pubkey not ours.
If we want to detect self invoices we need to find it in the swaps table but we don't expose such API.
I suggest not to check it and the sdk will return error if attempting to pay such invoice.

Reply:
I see, removed this false logic that tries to preemptively ignore self payments.